config/setup/xoscons.adb

```diff
@@ -353,61 +351,47 @@ procedure XOSCons is
          Info.Line_Number :=
            Integer (Parse_Int (Line (Index1 .. Index2 - 1), CNU).Abs_Value);
 
-         case Info.Kind is
```

Comment:
@TurboGit Although I understand your doubts with the case to if statement rule. This is an example I find definitely more readable.